src/xdainterface.cpp

```diff
@@ -183,6 +183,32 @@ void XdaInterface::registerPublishers()
     }
 }
 
+void XdaInterface::registerSubscribers(){
+    //Initialise GPS_Fix(NavSatFix) Subscriber
+    std::string fix_topic_name; 
+    ros::param::get("/xsens_mti_node/fix_topic_name", fix_topic_name);
```

Comment:
I suggest to remove the prefix for the param to follow the convention.
Also the param name should be intuitive, e.g., gps_fix_topic or gps_fix_source.